### DISCUSSION TITLE
Mania hardware-side lines

### DISCUSSION THREAD

**Discussion by krypton1101** [ppy/osu#30696] [Q&A] [answered]
These horizontal lines appear in the game and are related to the scroll speed, so they seem to move downward with notes.
They don't disappear when you quit the map or the entire app. I assume this is a hardware-side issue.

I note that it happens not in all maps and never noticed it to appear on not-4K maps.

Default scroll speed (7 or 8, not sure)
![IMG20241116201210](https://github.com/user-attachments/assets/9e49a70e-3a76-4878-bfdb-eb4c32775b4a)
Scroll speed 9
![IMG20241116201849](https://github.com/user-attachments/assets/01a9b7e1-7d6b-45ec-a034-3fbe57957527)
Scroll speed 6
![IMG20241116202130](https://github.com/user-attachments/assets/212d897d-5728-40ce-914e-35694b8b7cd7)
Lines haven't disappeared when quit osu!
![IMG20241116203445](https://github.com/user-attachments/assets/343a71b5-3a08-4f65-8b72-b8e2788233db)

My hardware:
- monitor: Asus VA24EHE (IPS, 1920x1080, 75Hz)
- GPU: AMD ATI Radeon 6700 XT
- CPU: AMD Ryzen 5 2600
- RAM: 16 GB

software:
- OS: Arch Linux (kernel: 6.11.7-arch1-1)
- DE: Hyprland

**Comment by peppy** [comment, marked as answer]
I can't see what you're talking about, but it's probably temporary screen burn-in.
